File: Bindings~/java/src/main/java/ro/altom/alttester/AltObjectProperty.java
package ro.altom.alttester;

public class AltObjectProperty {
    public String component;
    public String property;
    public String assembly;

    public AltObjectProperty(String assemblyName,String componentName, String propertyName) {
        assembly=assemblyName;
        component = componentName;
        property = propertyName;
    }

    public void setAssembly(String assembly){
        this.assembly = assembly;
    }
}

File: Assets/AltTester/Runtime/AltDriver/Common/AltObjectProperty.cs
/*
    Copyright(C) 2026 Altom Consulting
*/

namespace AltTester.AltTesterSDK.Driver
{
    public struct AltObjectProperty
    {
        public string Component;
        public string Property;
        public string Assembly;

        public AltObjectProperty(string component = "", string property = "") :
            this(component, property, null)
        { }

        public AltObjectProperty(string component, string property, string assembly)
        {
            Component = component;
            Property = property;
            Assembly = assembly;
        }
    }
}
